### ISSUE TITLE
Snakemake integration

### ISSUE THREAD

**Issue by johanneskoester** [tycho-kirchner/shournal#3]
It would be great if shournal could optionally output as much information for a particular query as possible via a --json flag. There is no live support with writing to a file or something like that needed. What I envision is a use like this:

```bash
shournal --query --wfile foo --json | snakemake-skeleton --shournal >> Snakefile
```

**Comment by tycho-kirchner** [comment]
Thanks for this suggestion, which I'm going to implement very soon. Right now I am adding functionality to add support for additionally logging arbitrary read-events (with hash) without storing the files to disk which is currently (shournal version <= 2.0) not possible. I guess that feature could also be interesting for snakemake with regard to the generation of input-rules.

**Comment by tycho-kirchner** [comment]
I have just released shournal version 2.1 which now supports json-output of commands and file-events:
`--query --output-format json`. If you need more information, please let me know.

**Comment by tycho-kirchner** [comment]
Closing this now. Just as a reference: The shournal-snakemake integration may be found
[here](https://github.com/snakemake/shournal-to-snakemake).
